public/app/core/components/OptionsUI/registry.tsx

```diff
@@ -280,6 +280,7 @@ export const getAllStandardFieldConfigs = () => {
     id: 'decimals',
     path: 'decimals',
     name: 'Decimals',
+    description: 'Leave empty to calculate based on the value',
```

Comment:
Do we really need this? by default it's empty, and the default (empty) placeholder says "auto"

Reply:
I mean this description does not really add any descriptive power other than saying it's automatic (and automatic I think implies it's based on the value) otherwise it would just be fixed

Reply:
Yeah, fair. I wanted to see if there was something we could include to help clarify a part of the confusion from this https://github.com/grafana/support-escalations/issues/3553. 

I dont think it’s clear what empty/auto means, and I think it’s reasonable to incorrectly assume that not specifying a value here will give you full precision.

Reply:
If i could concisely describe what auto is (e.g. "Leave blank to round to 3 significant places"), I would, but im not 100% sure exactly what it is 😅

Reply:
"Leave blank to round automatically"?

Reply:
see also, https://github.com/grafana/grafana/issues/53331#issuecomment-1220698029

Reply:
auto = grafana decides what happens to your number. Please ```git pull grafana``` and serach code base to see what auto does 

I kid, I kid! But I am trying to make the point that your basic grafana user has no clue what the heck happened to my trailing decimal numbers. 😉

Reply:
@torkelo Is the documentation for this correct? https://grafana.com/docs/grafana/latest/panels/configure-standard-options/#decimals 


> #### Decimals
> Number of decimals to render value with. Leave empty for Grafana to use the number of decimals provided by the data source.

"use the number of decimals provided by the data source" seems incorrect?

Reply:
@joshhunt yes, that is incorrect

Reply:
@joshhunt , taking a shot here to see if we get any closer to desired wording. "Specify the number of decimals Grafana includes in the rendered value. If you leave this field blank, Grafana automatically truncates the number of decimals based on the value. For example...(add one or two examples of what Grafana does to a specific value)."

I can continue helping to tweak the wording based on the example.